src/commands/adivinhar.js

```diff
@@ -85,23 +85,25 @@ async function sendGameMessageAndResults(interaction) {
 			await collectedMessage.message.delete();
 		}, 300);
 
+		const word = collectedMessage.message.content.normalize('NFKD').replace(/\p{Diacritic}/gu, '');
```

Comment:
It's not needed use `collectedMessage.message.content`, since `collectedMessage` const already has a property called `content`, that is a `Message.content` filtered.

Reply:
Right. I saw it in the "cancelar" check and thought it was the way to go :)